### PR TITLE
Add Family Mediators API staging namespace

### PR DESCRIPTION
**Pull request by zheileman**

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "${var.aws_region}"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
```
Comment (razvan-moj) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/ecr.tf]: ref=2.0 please, see https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+############################################
+# Family Mediators API RDS (postgres engine)
+############################################
+
+module "rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=2.1"
```
Comment (razvan-moj) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf]: ref=3.0

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/ecr.tf (hunk 1 above)
Comment (zheileman) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/ecr.tf]: I knew about the new release, but I had the doubt if I don't require the new functionality in release 2.0, could I continue using release 1.0?

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf (hunk 2 above)
Comment (zheileman) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf]: Yep, I will upgrade this to release 3.0

Comment (razvan-moj) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf]: yes, was just making sure - approved

Comment (razvan-moj) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf]: np, a small edit shouldn't require another approval

**Pull request merged**